### ISSUE TITLE
Bug: creator label formatting

### ISSUE THREAD

**Issue by bwiernik** [Juris-M/citeproc-js#93]
I am working with this style:
https://gist.github.com/bwiernik/f33d5ebf8793fbe663f826d478298c76

In the `"format"` macro, I use `names` to call `reviewed-author`:
```
<names variable="reviewed-author">
              <label form="verb-short" suffix=" "/>
              <name and="symbol" initialize-with=". " delimiter=", "/>
</names>
```

The expected output for this item:
```
[
	{
		"id": "http://zotero.org/groups/2205533/items/IJBGS4BQ",
		"type": "post-weblog",
		"title": "The future for Deaf individuals is not that bleak",
		"URL": "http://www.personalityresearch.org/papers/hall.html",
		"note": "reviewed-title: Decrease of Deaf potential in a mainstreamed environment\ngenre: Peer commentary\nreviewed-author: Hall || W.C.",
		"language": "en",
		"author": [
			{
				"family": "Wolf",
				"given": "K.S."
			}
		],
		"issued": {
			"date-parts": [
				[
					"2005"
				]
			]
		}
	}
]
```

is:
Wolf, K. S. (2005). The future for Deaf individuals is not that bleak [Peer commentary, _Decrease of Deaf potential in a mainstreamed environment_, by W. C. Hall]. Retrieved from http://www.personalityresearch.org/papers/hall.html

However, what renders instead is:
Wolf, K. S. (2005). The future for Deaf individuals is not that bleak [Peer commentary, _Decrease of Deaf potential in a mainstreamed environment_, W. C. Hallby ]. Retrieved from http://www.personalityresearch.org/papers/hall.html

With the `label` coming after `name`, even though it is specified to come before. If I move the `names` element out of the `"format"` macro (e.g., directly into the bibliography layout or into the `"title"` macro), it renders correctly. If I change `reviewed-author` to another creator variable, the formatting of `label` still ignores the formatting specified here and instead uses the formatting given elsewhere in the style for that variable (or following the `name` if it is not specified elsewhere). 

The cause is that `<text macro="format">` is included in the `substitute` list of the `"author-short"` macro. If I delete the three instances of `<text macro="format">` from the `"author-short"` macro (all three must be deleted, even the two in conditional branches that do not apply to the item), the `label` formatting is rendered correctly.

@adam3smith

**Comment by fbennett** [comment]
Thanks for this report. I found the problem, and I've fixed it in local code. A repaired version of the processor will be released when I've worked through a small backlog of fault reports.

**Comment by bwiernik** [comment]
Great! (And welcome back!)

**Comment by fbennett** [comment]
In the meantime, I've finally started work on that chair caning project that I mentioned long ago. Something more to sit on. :-)

**Comment by fbennett** [comment]
Fixed in tagged release 1.1.211.
